packages/karma-webkit-launcher-nimble/README.md

```diff
@@ -0,0 +1,5 @@
+# karma-webkit-launcher-nimble
+
+## Overview
+
+Forked and slightly modified version of [karma-webkit-launcher](https://github.com/google/karma-webkit-launcher/tree/main). See comment at the top of `karma-webkit-launcher.js` for details.
```

Comment:
Is this something we can upstream? I'm not particularly excited to fork and maintain all this

Reply:
their docs talk alot about playwright so I suspect they are interested in it working well

Reply:
I wonder if they'll be satisfied with a purely empirical justification, or if they'll want to better understand why this is happening. I can see what they say, though. On the other hand, that repo has had 38 total commits in the 3 years of its history, so I bet we'd be fine never touching these three files again.

Reply:
yea thinks it's worth creating an issue discussing the problem in detail and opening an example PR upstream.

Reply:
Dunno the details but similar sounding issues seem to hit them before https://github.com/google/karma-webkit-launcher/issues/5

Reply:
Created [issue](https://github.com/google/karma-webkit-launcher/issues/12) and [PR](https://github.com/google/karma-webkit-launcher/pull/13)

Reply:
FYI JIC looks like the PR is blocked on signing the Google CLA: https://github.com/google/karma-webkit-launcher/pull/13#issuecomment-2207649818

Reply:
May be worth pinging the person who self-assigned the issue that the CLA has been signed on the PR and it's ready to review. But even before that, just to confirm, did you test their workflows in your own fork? I don't see any actions run: https://github.com/m-akinc/karma-webkit-launcher/actions
Can help shorten review cycles to make sure their tests / workflows will pass.

Reply:
I've now run the workflows, and they [all pass](https://github.com/m-akinc/karma-webkit-launcher/actions) except the Safari one, which they've [disabled on their repo](https://github.com/google/karma-webkit-launcher/actions/workflows/safari_tests.yml) (presumably because it's broken). I'll ping the guy.

Reply:
Wow, it was merged immediately without feedback or questions and already released! I'll drop the forked copy and update our dependency.